tests/plugins/test_store_metadata.py

```diff
@@ -288,6 +289,40 @@ def test_metadata_plugin(tmpdir, br_annotations, expected_br_annotations,
         assert json.loads(annotations['media-types']) == expected_pulp_pull_results
 
 
+@pytest.mark.parametrize(('res'), (
+    {
+        'filesystem-koji-task-id': 'example-fs-taskid',
+        'base-image-id': 'foobar'
+    },
+    {
+        'base-image-id': 'foobar'
+    },
+    {}
+))
+def test_koji_filesystem_label(res):
+    workflow = prepare()
+    workflow.prebuild_results = {
+        PLUGIN_ADD_FILESYSTEM_KEY: res
+    }
+    runner = ExitPluginsRunner(
+        None,
+        workflow,
+        [{
+            'name': StoreMetadataInOSv3Plugin.key,
+            "args": {
+                "url": "http://example.com/"
+            }
+        }]
+    )
+    output = runner.run()
+    labels = output[StoreMetadataInOSv3Plugin.key]["labels"]
+
+    if 'filesystem-koji-task-id' in res:
+        assert 'filesystem-koji-task-id' in labels
+        assert labels['filesystem-koji-task-id'] == 'example-fs-taskid'
+
```

Comment:
We don't assert anything if `'filesystem-koji-task-id' not in res`. Are we just making sure the plugin copes with it OK? Maybe we should assert that no 'filesystem-koji-task-id' label is added.